<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/prefix1/main_icon.png</normaloff>:/new/prefix1/main_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="creatButtons">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>Create a class</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionChange_path_to_save"/>
    <addaction name="actionDelete_all_class"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChange_path_to_save">
   <property name="text">
    <string>Change path to save</string>
   </property>
  </action>
  <action name="actionDelete_all_class">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/delete_all.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Delete all class</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <selectedoff>:/new/prefix1/exit.png</selectedoff>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
